manual/guides/cyclus.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: manual
+title: Cyclus
+subtitle: "Cyclus - Creating your Ubuntu Environment"
+permalink: /manual/guides/cyclus
+---
+
+## Step 0. Clean Install Ubuntu
+Make sure your Ubuntu environment is clean to ensure the script works. 
+This is most easily done by performing a fresh install of Ubuntu. Otherwise,
+check that previous installations of cyclus have been removed, and the 
+bashrc file is in the default state. 
```

Comment:
Should there be a guide or link to instructions to how to clean install ubuntu?

Reply:
We could hyperlink to [here](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop#0)